docs/tutorials/13_quantum_random_access_optimizer.ipynb

```diff
@@ -0,0 +1,841 @@
+    "2. **Rounding Scheme**: To map the ground state results back to a solution for the original problem, we specify a rounding scheme. By default, the `SemideterministicRounding` is used, but alternative scheme, `MagicRounding`, is also available."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit.algorithms.optimizers import COBYLA\n",
+    "from qiskit.algorithms.minimum_eigensolvers import VQE\n",
+    "from qiskit.circuit.library import RealAmplitudes\n",
+    "from qiskit.primitives import Estimator\n",
+    "\n",
+    "from qiskit_optimization.algorithms.qrao import (\n",
+    "    QuantumRandomAccessOptimizer,\n",
+    "    SemideterministicRounding,\n",
+    ")\n",
+    "\n",
+    "\n",
+    "# Prepare the VQE algorithm\n",
+    "ansatz = RealAmplitudes(2)\n",
+    "vqe = VQE(\n",
+    "    ansatz=ansatz,\n",
+    "    optimizer=COBYLA(),\n",
+    "    estimator=Estimator(),\n",
+    ")\n",
+    "\n",
+    "# Use semi-deterministic rounding, known as \"Pauli rounding\"\n",
+    "# in https://arxiv.org/pdf/2111.03167v2.pdf\n",
+    "# (This is the default if no rounding scheme is specified.)\n",
+    "semidterministic_rounding = SemideterministicRounding()\n",
+    "\n",
+    "# Construct the optimizer\n",
+    "qrao = QuantumRandomAccessOptimizer(min_eigen_solver=vqe, rounding_scheme=semidterministic_rounding)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, we move forward with solving the problem by invoking the `solve()` method. It's important to note that when calling `solve()`, we pass the `problem` itself as an argument. Although we previously used `encode()` in `QuantumRandomAccessEncoding` to provide a clear understanding of the flow, `solve(problem)` automatically encodes the problem internally using `QuantumRandomAccessEncoding`. This provides a streamlined and simplified workflow that eliminates the need for explicit encoding steps.\n",
+    "\n",
+    "The result is provides us as a `QuantumRandomAccessOptimizationResult`.\n",
+    "The `x` contains the binary values representing the best solution found, while the `fval` contains the corresponding objective value.\n",
+    "\n",
+    "The `relaxed_fval` provides the expectation value of the relaxed Hamiltonian, adjusted to be in the units of the original optimization problem. For maximization problems, the best possible relaxed function value will always be greater than or equal to the best possible function value of the original problem. In practice, this often holds true for the best found value and best found function value as well."
```

Comment:
```suggestion
    "The `relaxed_fval` provides the expectation value of the relaxed Hamiltonian, adjusted to be in the units of the original optimization problem. For maximization problems, the best possible relaxed function value will always be greater than or equal to the best possible objective function value of the original problem. In practice, this often holds true for the best found value and best found objective function value as well."
```

Reply:
Just "function" or "function value" is ambiguous. Need to mention "objective function" or "objective function value".